src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -1094,6 +1094,11 @@
         </remarks>
       </Comments>
     </Property>
+    <Property Name="WasTargetTyped" Type="bool">
```

Comment:
><Property Name="WasTargetTyped" Type="bool"> [](start = 4, length = 44)

Why do we need this information in the node? This is not about semantics and it can easily be obtained from syntax. I don't think we have this info for array creation and other target typed operations.